test/commands.lisp

```diff
@@ -0,0 +1,26 @@
+(in-package #:lispkit-test)
+
+
+(5am:in-suite commands)
+
+(5am:test test-command-p
+  (lispkit:defcommand test-command ()
+    "Test command")
+  (multiple-value-bind (value presentp)
+      (lispkit::command-p "test-command")
+    (5am:is-true (eq presentp t))))
+
+(5am:test test-run-named-command
+  (lispkit:defcommand test-command (browser)
+    "Test command"
+    (eq browser "fixture"))
```

Comment:
I don't believe the CL standard requires implementations to translate two instances of the same literal string into  the same object, so this test can fail without there actually being a problem. `EQ` is not appropriate for strings.